app/src/main/java/br/com/catbag/gifreduxsample/models/AppState.java

```diff
@@ -19,4 +19,9 @@
         return gifs;
     }
 
+    @Value.Default
+    public boolean getHasMoreGifs() {
+        return true;
```

Comment:
Its better to be false, and on activity start a load verifies that fetch was sucessful, one list loaded and hasMoreGifs...because default being true we make a assumption that servers has a page to be fetch...